br/pkg/lightning/mydump/loader.go

```diff
@@ -290,8 +291,22 @@ func (s *mdLoaderSetup) listFiles(ctx context.Context, store storage.ExternalSto
 			return nil
 		}
 
+		schemaName, err := url.QueryUnescape(res.Schema)
```

Comment:
How can we tell if we need to unscape the schema? (`db%22` is also a valid database name)

Reply:
I think there is not easy to precisely check whether the input file name is escaped. This behavior is similar to our default file routint rule with a file name like `my.db.my.table.001.sql`, we don't know with `.` is the separator between dbname and tablename.